hackathon_site/dashboard/frontend/src/slices/hardware/cartSlice.ts

```diff
@@ -1,4 +1,62 @@
+import {
+    createEntityAdapter,
+    createSelector,
+    createSlice,
+    PayloadAction,
+} from "@reduxjs/toolkit";
+import { RootState } from "slices/store";
+
 export interface CartItem {
     hardware_id: number;
     quantity: number;
 }
+
+interface CartExtraState {
+    isLoading: boolean;
```

Comment:
@grahamhoyes @Mustaballer why do we need isLoading for the cart?

Reply:
> @grahamhoyes @Mustaballer why do we need isLoading for the cart?

I believe it's going to be for when we submit the requests to place the order.

Reply:
ahh okay got it

Reply:
Yup exactly, unless we decide to put that thunk in an Order store at which case we won't need these extra fields.